vpdq/cpp/hashing/filehasher.h

```diff
@@ -33,6 +36,7 @@ bool hashVideoFile(
     const int secondsPerHash,
     const int width,
     const int height,
+    const double duration_in_sec,
```

Comment:
nit: format inconsistency with rest of arguments `lowerCamelCase` vs `lower_snake_case`